.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  NODE_VERSION: '14'
+  NODE_VERSION: '16'
```

Comment:
does this mean we are no longer using `14.18.2` locally ?

Reply:
Considering we are not actively using Node code ourselves (beside the custom linter rules) and that ember-cli is compatible there should not be any issues.

But i remember having issues too just not in this project, I guess we'll see them down the road while migrating to pnpm (which requires Node >= 16)

In any case, this doesn't force us to migrate everything till we move to pnpm. (IMO we should go even further and move to Node 18 but that's too much a change and one thing a time 😅 )